### PR TITLE
Feat: added login and signup pages

### PR DESCRIPTION
**Pull request by coldpigli**
Login Page

image

Sign-Up Page

image

### CODE REVIEW

**Review by shubhamkumar648 — APPROVED** [review]
Good use of the class name and utils class. reaming code looks clean , you can merge

**Review by rahulcs754 — APPROVED** [review]
looks good  and  clean code

File: routes/login-page.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        <div class="input-wrapper gap-d20">
+            <label class="input-name" for="input">Name</label>
+            <input class="input-box" type="text" placeholder="Enter your email">
+        </div>
+        <div class="input-wrapper gap-d20">
+            <label class="input-name" for="password">Password</label>
+            <input class="input-box" type="password" value="passwordplaceholder" placeholder="Enter your name">
+        </div>
+        <p class="paragraph2 txt-gray gap-d30"> <a href="#">Forgot Password?</a></p>
+        <button class="btn btn-primary gap-d20">
```
Comment (kjais1720) [inline routes/login-page.html:26]: It's not very clear what does the `d` signify in the gap class?

**Review by kjais1720 — APPROVED** [review]
Looks good overall, just a small comment (mentioned above)

File: routes/login-page.html (hunk 1 above)
Comment (coldpigli) [reply routes/login-page.html:26]: Thank you for pointing it out. This comes from a utility class from my Component Library.
Have noted this down. Will change the name to something more intuitive when I refactor the library. Thanks

**Pull request merged**